lib/api/include/irods/apiTable.hpp

```diff
@@ -1269,6 +1271,20 @@ static irods::apidef_t client_api_table_inp[] = {
         "api_get_library_features", irods::clearInStruct_noop, irods::clearOutStruct_noop,
         (funcPtr) CALL_GET_LIBRARY_FEATURES_OUT
     },
+    {
+        GET_RESCINFO_FOR_PUT_AN, RODS_API_VERSION, REMOTE_USER_AUTH, REMOTE_USER_AUTH,
+        "DataObjInp_PI", 0, "STR_PI", 0,
+        boost::any(std::function<int(rsComm_t*,dataObjInp_t*,char**)>(RS_GET_RESCINFO_FOR_PUT)),
+        "api_get_rescinfo_for_put", clearDataObjInp, irods::clearOutStruct_noop,
+        (funcPtr)CALL_DATAOBJINP_CHAROUT
+    },
+    {
+        GET_RESCINFO_FOR_GET_AN, RODS_API_VERSION, REMOTE_USER_AUTH, REMOTE_USER_AUTH,
```

Comment:
I think `REMOTE_USER_AUTH` should be `LOCAL_USER_AUTH`.

Reply:
See conversation at https://github.com/irods/irods/pull/7255#discussion_r1304583661